tests/lcs/core/strings/filter.livecodescript

```diff
@@ -0,0 +1,213 @@
+﻿script "CoreStringFilter"
+/*
+Copyright (C) 2015 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+/*
+	Filter wildcard special characters:
+	* - matches any number of occurrences of any character
+	? - matches a single occurrence of any character
+	[abc] - matches any of a, b or c
+	[a-m] - matches any lowercase character in the first half of the alphabet
+	[!<string>] - matches iff [<string>] does not match
```

Comment:
There was a 'regex reference card' somewhere in the IDE at one point... We should probably try and find that and put into the dictionary / guides in a more appropriate place!